src/lib/y2network/interfaces_collection.rb

```diff
@@ -70,6 +71,14 @@ def by_name(name)
       end
     end
 
+    # Returns list of interfaces of given type
+    #
+    # @param type [String] device type name ("eth", "br", ...)
```

Comment:
We should use an `InterfaceType` instance, although we could be able to handle strings/symbols too.